config/schema_oracle.sql

```diff
@@ -50,6 +50,7 @@ user_id VARCHAR(99) NOT NULL,
 state VARCHAR(99),
 access_token VARCHAR(99),
 refresh_token VARCHAR(99),
+refresh_token_createdon TIMESTAMP,
```

Comment:
little bit buzz on this... There should be _ in between created_on. Sample semantics apply else where in the code